data-loader/core/src/main/java/com/scalar/db/dataloader/core/dataimport/processor/JsonImportProcessor.java

```diff
@@ -45,6 +44,7 @@ public class JsonImportProcessor extends ImportProcessor {
 
   private static final DataLoaderObjectMapper OBJECT_MAPPER = new DataLoaderObjectMapper();
   private static final AtomicInteger dataChunkIdCounter = new AtomicInteger(0);
+  private static final int MAX_QUEUE_SIZE = 10;
```

Comment:
This queue size might be too less when using a large number of cores like 32 cores. I guess this limitation is to prevent OOM. If so, 256 or 512 should be okay. Maybe it would be great if this is configurable.

I think you can decide whether to change this after conducting some performance tests with various numbers of cores.

Reply:
@komamitsu san,
Thank you for the suggestion, I have now set it to 256. We are working on performance tests once that is completed I will update this accordingly.

Reply:
@komamitsu san,
I have made this value configurable. I will set a suitable value once the performance testing is completed. For now it is set to `256`.